src/Symfony/Component/Routing/Matcher/Dumper/ApacheMatcherDumper.php

```diff
@@ -94,7 +94,7 @@ public function dump(array $options = array())
         }
         if (0 < count($methodVars)) {
             $rule = array('# 405 Method Not Allowed');
-            $methodVars = array_values(array_unique($methodVars));
+            $methodVars = array_keys(array_count_values($methodVars));
```

Comment:
this one is also a warmup code path (and not even running by default, and deleted in 3.x)